### PR TITLE
Task 52

### PR DESCRIPTION
**Pull request by LukaBis**
User can see all his events in one table. User can create new event with event form.

### CODE REVIEW

File: frontend/src/components/EventForm/EventForm.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+                label="Description"
+                className="event-form-input"
+                sx={{ mt: 3 }}
+                variant="standard"
+                value={description}
+                onChange={(e) => setDescription(e.target.value)}
+            />
+            <br />
+            <label style={{ marginTop: '2rem' }}>Date: </label>
+            <input
```
Comment (Luka8st) [inline frontend/src/components/EventForm/EventForm.js:61]: Past dates should be unavailable

File: frontend/src/components/EventForm/EventForm.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+                variant="contained"
+                sx={{ mt: 5, width: '150px' }}
+                onClick={handleSubmit}
+            >
+                CREATE
+            </Button>
+
+            {successAlert ? (
+                    <Alert severity="success" sx={{ mt: 4 }}>
+                        Successful update!
```
Comment (Luka8st) [inline frontend/src/components/EventForm/EventForm.js:94]: Events aren't being updated but created, so the message should be changed

File: frontend/src/components/EventForm/EventForm.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    const [title, setTitle] = useState('');
+    const [description, setDescription] = useState('');
+    const [date, setDate] = useState("");
+    const [time, setTime] = useState("");
+    const [image, setImage] = useState("");
+
+    const [successAlert, setSuccessAlert] = useState(false);
+    const [errorAlert, setErrorAlert] = useState(null);
+
+    const handleSubmit = () => {
```
Comment (Luka8st) [inline frontend/src/components/EventForm/EventForm.js:19]: Input fields and submit button should be disabled while the request is being processed

File: frontend/src/components/EventForm/EventForm.js [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+        console.log(title, description, date, time, image)
+        createNewEvent(title, description, date, time, image)
+        .then(res => {
+            if(res == 'OK') {
```
Comment (Luka8st) [inline frontend/src/components/EventForm/EventForm.js:23]: After a successful request, the fields should be reset

File: frontend/src/components/EventForm/EventForm.js (hunk 1 above)
Comment (LukaBis) [reply frontend/src/components/EventForm/EventForm.js:61]: Inside backend/app/Http/Requests/StoreEventRequest.php and UpdateEventRequest.php I added rule that date has to be **after_or_equal:today**

File: frontend/src/components/EventForm/EventForm.js (hunk 2 above)
Comment (LukaBis) [reply frontend/src/components/EventForm/EventForm.js]: Changed to **Successfully created!**

File: frontend/src/components/EventForm/EventForm.js (hunk 3 above)
Comment (LukaBis) [reply frontend/src/components/EventForm/EventForm.js:19]: All input fields are no disabled until we receive the response.

File: frontend/src/components/EventForm/EventForm.js (hunk 4 above)
Comment (LukaBis) [reply frontend/src/components/EventForm/EventForm.js:32]: Done

**Review by Luka8st — APPROVED** [review]
Everything works well

**Pull request merged**